servicetalk-buffer-netty/build.gradle

```diff
@@ -16,6 +16,10 @@
 
 apply plugin: "io.servicetalk.servicetalk-gradle-plugin-internal-library"
 
+test {
```

Comment:
Same as `buffer-api` this is not needed in here since its provided from ST plugin.